Underscore/USArrayWrapper.m

```diff
@@ -168,8 +168,9 @@ - (USArrayWrapper *)shuffle
 {
     NSMutableArray *result = [self.array mutableCopy];
 
+    NSInteger max = result.count;
```

Comment:
Can we make this variable `u_int32_t` and move the cast here? Also, can you `NSAssert` that count must be less than or equal to `UINT32_MAX`? Unlikely I know, but [stranger things have happened](http://googleresearch.blogspot.de/2006/06/extra-extra-read-all-about-it-nearly.html).